.github/workflows/sycl_nightly.yml

```
@@ -48,6 +54,41 @@ jobs:
       retention-days: 90
       lts_matrix: ${{ needs.test_matrix.outputs.lts_wn_matrix }}
 
+      # We use the build for the release, override its name.
+      artifact_archive_name: sycl_windows.tar.gz
```

```suggestion
      artifact_archive_name: dpcpp.win.tgz
```
I think usually tar is not used on windows. Can we just use (self-extracting?) zip or Windows installer to make handling easier for Windows users?